### DISCUSSION TITLE
Password reset

### DISCUSSION THREAD

**Discussion by som3thingwitty** [LiveHelperChat/livehelperchat#2062] [General]
I'm having an issue with the password reset. I get the email follow the link, it says it's successful on reset but the old password still works
I check the DB and there are no changes there. 
I can update the password when logged in, it's just not working when running the password recover
I'm on current version 4.31
Any help?

**Comment by remdex** [comment]
Can it be you have two users with same username or same e-mail? I just tried on my dev and all works as expected.

**Reply by som3thingwitty** [reply]
This was it, checked the DB and I had same email for two accounts. Thanks!
